app/controllers/proxy_rule_shared_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# This module is used to share behave between the 2 different Mapping rules controller that we have
+# (1 for BackendApi and another one to the Service).
```

Comment:
```suggestion
# (1 for BackendApi and another one to the Proxy).
```

Reply:
This one is tricky as a `ProxyRule` either belong to a `BackendApi` or to a `Proxy`, but never to a `Service`. Nevertheless the controller is right below service. Confusing.